unity3d/Assets/Swrve/SwrveSDK/Messaging/SwrveConversationCampaign.cs

```diff
@@ -46,8 +46,11 @@ public SwrveConversation GetConversationForEvent(string triggerEvent, IDictionar
 
         if (checkCampaignLimits (triggerEvent, payload, qaUser)) {
             SwrveLog.Log (string.Format ("[{0}] {1} matches a trigger in {2}", this, triggerEvent, Id));
-
-            return Conversation;
+            if (AreAssetsReady ()) {
```

Comment:
It's for a long standing bug with messages from Session Start, they've always had a different path for selection, and that long standing issue Louise used have with IAMs not showing at start was due to this.  It wouldn't check if a message was ready, in the same way as normal.

Needs a big enough refactor overall for choosing, it's gotten quite messy, but I was thinking we'd do that after the architecture review for 5.0.